### ISSUE TITLE
Feature Request: Merging Multiple Pokedexes

### ISSUE THREAD

**Issue by kaptanultros** [pokedextracker/pokedextracker.com#472]
Currently, Scarlet and Violet are not compatible with Home, and if I wanted to update my "Home" Pokedex with anything from Scarlet and Violet (including any returning Pokemon), I need to manually go into both and cross-reference them.

I'd love to see a "Merge" feature that would let me mark everything marked as "caught" in one Pokedex as "caught" in a separate Pokedex.

For example, if I opened up my "Scarlet" Pokedex, and clicked "Merge", there would be a prompt for the Pokedex to update with the information from my "Scarlet" Pokedex, and selecting my "Home" Pokedex would update it with "caught" status for everything marked as such in "Scarlet".

**Comment by danwdart** [comment]
Yes, agreed, this is especially difficult since I have to make a new dex for the expansions when I already had one, and can't just convert it, it seems.
